Mathlib/MeasureTheory/Measure/Haar/Quotient.lean

```diff
@@ -85,7 +85,7 @@ lemma MeasureTheory.QuotientMeasureEqMeasurePreimage.smulInvariantMeasure_quotie
     obtain ⟨𝓕, h𝓕⟩ := hasFun.ExistsIsFundamentalDomain
     have h𝓕_translate_fundom : IsFundamentalDomain Γ.op (g • 𝓕) ν :=
       h𝓕.smul_of_comm g
-    rw [h𝓕.projection_respects_measure_apply (μ := μ)
+    erw [h𝓕.projection_respects_measure_apply (μ := μ)
       (meas_π (measurableSet_preimage (measurable_const_smul g) hA)),
       h𝓕_translate_fundom.projection_respects_measure_apply (μ := μ) hA]
```

Comment:
If only one of these needs an `erw`, can you separate it into a `rw` and an `erw`? Same on the previous one.